docs/azure_jumpstart_ag/manufacturing/contoso_motors/arc_monitoring_servers/_index.md

```diff
@@ -9,4 +9,104 @@ serviceOrPlatform: INFRASTRUCTURE
+
+## Overview
+
+Infrastructure observability is crucial for Contoso Motors to understand the performance and health of their Azure Arc-enabled servers. This is where [Azure Monitor](https://learn.microsoft.com/azure/cloud-adoption-framework/scenarios/hybrid/arc-enabled-servers/eslz-management-and-monitoring-arc-server) comes into play, playing a vital role in offering visibility into every facet of their Azure Arc-enabled servers ecosystem.
+
+Azure Monitor empowers Contoso with the capability to monitor and gather telemetry data from their Azure Arc-enabled servers. It serves as a central hub, providing near real-time insights into server performance, health, and resource utilization. Azure Monitor offers a comprehensive view of the entire infrastructure, ensuring proactive identification and resolution of potential issues.
+
+## Enable and configure Azure Monitor
+
+Azure Monitor can collect data directly from your Arc-enabled servers into a Log Analytics workspace for detailed analysis and correlation. It requires installing the Azure Monitor Agent (AMA) VM extension in your Azure Arc-enabled servers, enabling VM insights to collect data from your machines.
+
+As part of the automated deployment, an Azure Policy monitoring initiative and a Data Collection Rule (DCR) are deployed. They allow collecting monitoring data from your Azure Arc-enabled servers.
+
+Follow these steps to verify that these required Azure Monitor artifacts have been successfully deployed:
+
+- In the top bar of the Azure portal, search for **policy** and click on **Policy**:
+
+    ![Screenshot of searching Azure Policy](./img/search_policy.png)
+
+- Click on **Assignments**. You will see the Azure Policy initiative **(Ag) Enable Azure Monitor for Hybrid VMs with AMA**. This initiative enables Azure Monitor for the hybrid virtual machines with AMA. It takes a Log Analytics workspace as and a Data Collection Rule (DCR) as parameters.
+
+    ![Screenshot of Azure Monitor initiative assignment Azure Policy](./img/azure_monitor_initiative.png)
+
+- The DCR is in charge of collecting monitoring data from the Azure Arc-enabled servers. In the top bar, search for **Data collection rules**:
+
+    ![Screenshot of searching Data Collection Rules](./img/search_dcr.png)
+
+- You will find the DCR that has been created to collect insights from the Azure Arc-enabled servers:
+
+    ![Screenshot of the Data Collection Rules](./img/dcr_vmi.png)
+
+- Click on the DCR. You will see the data sources collected, in this case, performance counters:
+
+    ![Screenshot of the DCR - Data sources](./img/dcr_datasources.png)
+
+## Azure Arc-enabled servers and Azure Monitor VMInsights Integration
+
+Now that we have checked that the required monitoring artifacts have been successfully enabled, it's time to leverage VMInsights. It monitors the performance of your Azure Arc-enabled servers by collecting the required data with AMA.
```

Comment:
We should use the same casing as https://learn.microsoft.com/en-us/azure/azure-monitor/vm/vminsights-overview
```suggestion
Now that we have checked that the required monitoring artifacts have been successfully enabled, it's time to leverage VM insights. It monitors the performance of your Azure Arc-enabled servers by collecting the required data with AMA.
```